function_test.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2021 Roger Chapman and the v8go contributors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package v8go_test
+
+import (
+	"testing"
+
+	"rogchap.com/v8go"
+)
+
+func TestFunctionCall(t *testing.T) {
+	t.Parallel()
+
+	ctx, err := v8go.NewContext()
+	failIf(t, err)
+	_, err = ctx.RunScript("function add(a, b) { return a + b; }", "")
+	failIf(t, err)
+	addValue, err := ctx.Global().Get("add")
+	failIf(t, err)
+	iso, _ := ctx.Isolate()
+
+	arg1, err := v8go.NewValue(iso, int32(1))
+	failIf(t, err)
+
+	fn, _ := addValue.AsFunction()
+	resultValue, err := fn.Call([]v8go.Valuer{arg1, arg1})
```

Comment:
This is looking great @robfig.
Sorry to be a pain, but looking at this I'm wondering if a variadic function would be more idiomatic? Then you could do:
```
fn.Call(arg1, arg2, etc)
```

Reply:
No problem at all -- agreed that it looks cleaner. Updated